spaceship/lib/spaceship/portal/device.rb

```diff
@@ -59,9 +59,10 @@ def factory(attrs)
         end
 
         # @param mac [Bool] Fetches Mac devices if true
+        # @param includeDisabled [Bool] Whether to include disable devices. false by default.
         # @return (Array) Returns all devices registered for this account
-        def all(mac: false)
-          client.devices(mac: mac).map { |device| self.factory(device) }
+        def all(mac: false, includeDisabled: false)
+          client.devices(mac: mac, includeDisabled: includeDisabled).map { |device| self.factory(device) }
```

Comment:
Sorry for that, just a habit. Fixed now. Don't think it should be highlighted by rubocop?